<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LblStatus[state = &quot;0&quot;] {
	background-color: rgb(237, 51, 59);
}

#LblStatus[state = &quot;1&quot;] {
	background-color: rgb(38, 162, 105);
}

#GrpInfo{
	background-color: rgb(192, 191, 188);
	color: rgb(0, 0, 0);
}

#LblPort {
color: rgb(0, 0, 0);
}

#LayConsole{
	background-color: rgb(192, 191, 188);
	color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GrpInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="LblPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpnPort">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnStart">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Server</string>
     </property>
    </widget>
    <widget class="QLabel" name="LblStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="state" stdset="0">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>258</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="LayConsole">
     <item>
      <widget class="QListWidget" name="LstConsole"/>
     </item>
     <item>
      <widget class="QPushButton" name="BtnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnClear</sender>
   <signal>clicked()</signal>
   <receiver>LstConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>739</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
